src/components/AppMapSettings.ts

```diff
@@ -18,14 +18,16 @@ export default class AppMapSettings extends Vue {
   s: Settings | null = null;
 
   optionsMapType = Object.freeze([
-    { value: 'Totk', text: 'Sky, Surface and Depths' },
+    { value: 'Any', text: 'Sky, Surface and Depths' },
```

Comment:
Looking at Radar, I don't think we should call this Any. Any suggests this is used for any map type, but in fact this option means "MainField or MinusField". Let's call this MainAndMinusField or something similarly explicit

Reply:
MainAndMinusField. Done